spec/app/domain/authentication/authn_k8s/validate_pod_request_spec.rb

```diff
@@ -80,7 +80,7 @@
       .to receive(:new)
             .and_return(validate_application_identity)
     allow(validate_application_identity).to receive(:call)
-                                          .and_return(true)
+                                              .and_return(true)
```

Comment:
More formatting.  In my recent work I've been doing:
```
allow(Authentication::AuthnK8s::ValidateApplicationIdentity).to(
  receive(:new).and_return(validate_application_identity)
)
allow(validate_application_identity).to(
  receive(:call).and_return(true)
)
```
Wdyt?

Reply:
It does look better but we have the current style 100+ places. Let's address this out of the scope of this PR.